core/iwasm/libraries/simde/simde.cmake

```diff
@@ -4,9 +4,12 @@
 
 set (LIB_SIMDE_DIR ${CMAKE_CURRENT_LIST_DIR})
 
-if (WAMR_BUILD_TARGET MATCHES "AARCH64.*" OR "ARM.*")
+if (WAMR_BUILD_TARGET MATCHES "AARCH64.*" OR WAMR_BUILD_TARGET MATCHES "ARM.*")
```

Comment:
I can see this file is conditionally enabled through a `WAMR_BUILD_LIB_SIMDE` flag. After thinking a bit about it, I'm not sure this is the best approach; `WAMR_BUILD_*` flags are public flags that are supposed to be set by the user; SIMDE is an implementation detail, and user shouldn't really know about it. 
I think we should be failing straight away if the user sets `WAMR_BUILD_SIMD` and there's no implementation of SIMD for their platform.